### DISCUSSION TITLE
【灌水】绳网怎么这么多诈骗的

### DISCUSSION THREAD

**Discussion by GanlinSama** [share121/inter-knot#1090] [其他]
如题，今天已经上了四次当了，治安局的反诈打击力度不太够啊😡（doge

**Comment by WORLD-executeME** [comment]
治安局官方号已经发帖回应这种问题了
https://www.bilibili.com/video/BV1qf42127cD/?spm_id_from=333.337.search-card.all.click&vd_source=55eda685f8ab5cd1fd1febdeb1c5e5a2

**Reply by ijog36** [reply]
哈哈，新型诈骗是吧

**Reply by Teacup1233** [reply]
严查治安局

**Comment by GanlinSama** [comment]
好好好，来骗，来偷袭我这个老同志，耗子尾汁吧(╯▔皿▔)╯
